csharp/GSDK_CSharp_Standard/GameserverSDK.cs

```diff
@@ -61,8 +55,7 @@ public static class GameserverSDK
         /// </returns>
         public static bool ReadyForPlayers()
         {
-            Task.WhenAll(_internalSdk.StartAsync())
-                .Wait();
+            _internalSdk.Start();
```

Comment:
>Start [](start = 25, length = 5)

this class calls Start() in every method, and Start() internally skip if it is called more than once. Essentially Start() is an initialization method. Is it possible to merge it with InternalSdk constructor or make a factory method for it?

Reply:
Yeah, i tried adding a static constructor where I could call start. However, the problem is the argument to the Start() method, which is currently left to the caller (game server). We will need to make it a config value in app.config or something and then we can do this.
I spent sometime on making it static constructor and then decided that we can tackle that one separately. Also, like you pointed out, we don't really need the InternalSDK class. That's for another time as well.